models/dalec/R/write.configs.dalec.R

```diff
@@ -121,7 +121,7 @@ write.config.DALEC <- function(defaults, trait.values, settings, run.id) {
       sla <- default.param[which(default.param$cmdFlag == "SLA"),"val"] * 1000 #convert SLA to m2/kgC from m2/gC (dalec default)
     }
     
-    IC.pools <- PEcAn.data.land::prepare_pools(IC.path, constants = list(sla = sla))
+    IC.pools <- PEcAn.data.land::prepare.pools(IC.path, constants = list(sla = sla))
```

Comment:
It's the other way around. We _were_ originally using . but discovered that led to a lot of S3 conflicts and more recently have been switching to underscore, thus functions that use . are legacy and in need of updating.

@tonygardella could you fix the documentation ASAP so that the style guide is in line with our actual practices